src/quick-lint-js/translation-table-generated.h

```diff
@@ -1,12 +1,16 @@
 // Code generated by tools/compile-translations.go. DO NOT EDIT.
-// source po/*.po
+// source: po/*.po
 
 // Copyright (C) 2020  Matthew "strager" Glazar
 // See end of file for extended copyright information.
 
+
 #ifndef QUICK_LINT_JS_TRANSLATION_TABLE_GENERATED_H
 #define QUICK_LINT_JS_TRANSLATION_TABLE_GENERATED_H
 
+
+
+
```

Comment:
Must fix: These changes reappeared after I ran `./tools/update-translator-sources`. Fix the code generator.

Reply:
ok. I have fixed it. 

the blank lines was introduced  in `tools/compile-translations.go`